core/services/ocr2/plugins/ocr2keeper/evmregistry/v21/logprovider/recoverer.go

```diff
@@ -674,6 +677,7 @@ func (r *logRecoverer) addPending(payload ocr2keepers.UpkeepPayload) error {
 	if !exist {
 		r.pending = append(pending, payload)
```

Comment:
Use `prommetrics.AutomationRecovererPendingPayloads.Inc()` instead of  `prommetrics.AutomationRecovererPendingPayloads.Set(float64(len(r.pending)))`